applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -93,6 +93,12 @@ static void button_msg_sub_thread(void)
 					LOG_WRN("Failed to stop broadcast sink: %d", ret);
 				}
 			} else if (strm_state == STATE_PAUSED) {
+				ret = bt_mgmt_scan_start(0, 0, BT_MGMT_SCAN_TYPE_BROADCAST, NULL);
+				if (ret) {
+					LOG_WRN("Failed to start scanning for broadcaster: %d",
```

Comment:
LOG_ERR?

Reply:
Should the error be handled?

Reply:
I will do more check to see if there's a racing condition for trigger the scan multiple times.
Too bad there's no way to check if there's a scan on going.